packages/wonder-blocks-dropdown/components/dropdown.test.js

```diff
@@ -1,488 +1,158 @@
 //@flow
```

Comment:
I'll review these tests after you rebase this PR over a new PR that does just the renaming.